btrack/utils.py

```diff
@@ -344,7 +344,12 @@ def _cat_tracks_as_dict(tracks: list, properties: list):
     return data
 
 
-def tracks_to_napari(tracks: list, ndim: int = 3, replace_nan: bool = True):
+def tracks_to_napari(
+    tracks: list,
+    ndim: int = 3,
+    coords_axes: list = None,
```

Comment:
`Optional` comes from `typing`. Or could this be an empty list `[]` instead?

```suggestion
    coords_axes: Optional[list] = None,
```